src/utils/person.js

```diff
@@ -0,0 +1,11 @@
+module.exports.getAvatarUrl = function (user) {
+	if (user?.avatar_url) {
```

Comment:
I'm thinking that, instead of Elvis-ing our way through this, we should provide an early return if no `user` is found in `people`:

```suggestion
module.exports.getAvatarUrl = function (user) {
	if (!user) {
		return '/assets/avatar-fallback.jpg';
	}
	
	if (user.avatar_url) {
```

This keeps us from having to null-check throughout the rest of the logic. Maybe that's not so necessary, since we have so few checks in here anyways.

What do you think?

Reply:
i like it!